### ISSUE TITLE
Enable clipping the color scale using `perc` in `cr.pl.circular_projection`

### ISSUE THREAD

**Issue by Marius1311** [theislab/cellrank#481]
- [ ] Additional function parameters, change functionality or change defaults?
- [ ] New estimator in `cr.tl.estimators`?
- [ ] New kernel in `cr.tl.kernels`?
- [ ] New gene trend model in `cr.ul.models`?
- [ ] New plotting function in `cr.pl`?
- [x] Other?

See scVelo's `scatter` plot API: https://scvelo.readthedocs.io/scvelo.pl.scatter.html#scvelo.pl.scatter. Basically, the `perc` argument is a list like `[1, 99]`, which means that values above the 99th percentile will be clipped to the 99th percentile and values below the 1st percentile will be clipped to the 1st percentile. This is done just for plotting and eases visual interpretation in the presence of outliers.

**Comment by michalk8** [comment]
Also, there's a problem with colorbar and too many lineages.

**Comment by michalk8** [comment]
@Marius1311 re both issues:
- you can already pass `perc=[0, 95]` and it works
- for the clippng between label name and colorbar, there's `labeldistance=1.25`, try lowering it (below 1 will be inside the polygon.

Here's a result of:
```python
cr.pl.circular_projection(adata, "initial_size_unspliced", perc=(0, 95), labeldistance=2)
```
![index](https://user-images.githubusercontent.com/46717574/106648178-3a22fe80-6590-11eb-9ca0-4cc540f088ad.png)

I don't want to further expose `perc` in the signature, since it's quite large already, hope this helps - closing this.

**Comment by Marius1311** [comment]
Thanks @michalk8, `labeldistance` solved the issue. As to `perc`, I realised the parameter exists but I just had the feeling it makes no difference, hence wasn't propagated. However, we can leave this for now and I can check this in another example and open a new issue if it persists there.

**Comment by Marius1311** [comment]
Thanks @michalk8, `labeldistance` solved the issue. As to `perc`, I realised the parameter exists but I just had the feeling it makes no difference, hence wasn't propagated. However, we can leave this for now and I can check this in another example and open a new issue if it persists there.

**Comment by Marius1311** [comment]
mhm, working on the figure, I realised that I need to be able to turn off the colorbar, would it be possible to introduce this with a `colorbar=False` parameter (scVelo like)? Currently, this yields
```
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-66-6d1ab7557ab4> in <module>
----> 1 cr.pl.circular_projection(adata, keys=['priming_degree'], 
      2                           legend_loc='right', figsize=(45, 45), title='', s=70, labeldistance=1.2, colorbar=False)

~/Projects/cellrank/cellrank/pl/_circular_projection.py in circular_projection(adata, keys, backward, lineages, lineage_order, metric, normalize_by_mean, ncols, space, use_raw, text_kwargs, labeldistance, labelrot, show_edges, key_added, figsize, dpi, save, **kwargs)
    310         if set_lognorm:
    311             cbar = ax.collections[0].colorbar
--> 312             cax = cbar.locator.axis
    313             ticks = cax.minor.locator.tick_values(cbar.vmin, cbar.vmax)
    314             ticks = [ticks[0], ticks[len(ticks) // 2 + 1], ticks[-1]]

AttributeError: 'NoneType' object has no attribute 'locator'
```

**Comment by michalk8** [comment]
will be fixed in #482

**Comment by michalk8** [comment]
closed via #482

**Comment by Marius1311** [comment]
great, thanks!
